### PR TITLE
Add import/export of public keys #79

### PR DESCRIPTION
**Pull request by alessio**
Hi,

This is to address a feature requests I previously raised in #79.

Thanks for considering.

### CODE REVIEW

**Comment by adrianbrink** [comment]
Thank you very much for your contrib @alessio :-)

**Review by melekes — APPROVED** [review]

File: keys/keybase.go [hunk 1]
```diff
@@ -140,6 +159,26 @@ func (kb dbKeybase) Import(name string, armor string) (err error) {
 	return nil
 }
 
+// ExportPubKey imports ASCII-armored public keys.
```
Comment (ebuchman) [inline keys/keybase.go:162]: Import

**Review by ebuchman — APPROVED** [review]

File: keys/keybase.go (hunk 1 above)
Comment (ebuchman) [reply keys/keybase.go:162]: fixed on develop.

**Comment by ebuchman** [comment]
Thanks!

**Pull request merged**